container/src/app/login/login.component.ts

```diff
@@ -138,4 +151,16 @@ export class LoginComponent implements OnInit {
   public open() {
     const dialogRef = this.dialog.open(VideoComponent)
   }
+
+  mascotTailAnimation(): void {
+    let catsTail = document.getElementById('mascot-tail');
```

Comment:
should be const

Reply:
I fixed everything as you asked. Please check it again